test/initial_tfm.jl

```diff
@@ -0,0 +1,211 @@
+using StaticArrays, Interpolations, LinearAlgebra
+using Images, CoordinateTransformations, Rotations
+using RegisterQD
+
+g = 0.2:0.2:1.2
+gradcube = g .* reshape(g, 1, 6) .* reshape(g, 1, 1, 6)
+gradcube[1,4,2] = 0
+gradcube[4,5,2] = 0 #break up any potential rotational symmetry
+
+mxshift = (1,1,1);
+mxrot = (0.01,0.01,0.01);
+minwidth_rot = fill(0.002,3);
+
+EYE =Matrix(1.0*I, 3,3)
+
+@testset "initial_tfm improves rigid rotational alignment" begin
+    testimage1 = zeros(Float64,10,10,10)
+    testimage1 .= NaN
+    testimage1[3:8,3:8,3:8] = gradcube
+    testimage1 = centered(testimage1)
+
+    tform = RotXYZ(0.1,0.1,0.1)
+    mytform = AffineMap(tform, [0,0,0])
+    testimage2 = warp(testimage1, mytform, axes(testimage1))
+
+    # insufficient parameters + no initiat_tfm causes large misalignment
+
+    tformtest1, mm1 = qd_rigid(testimage2, testimage1, mxshift, mxrot; print_interval=typemax(Int)) #
+    @test !isapprox(1-mm1, 1)
+    @test !isapprox(tformtest1, mytform, atol = 0.1)
+
+    #initiat_tfm improves alignment
+
+    tformtest2, mm2 = qd_rigid(testimage2, testimage1, mxshift, mxrot; print_interval=typemax(Int), initial_tfm = mytform) #
+ #with the initial_tfm being the true tfm, this should give back the true rotation
+
+    @test isapprox(1-mm2, 1)
```

Comment:
```suggestion
    @test abs(mm2) < 1e-8
```
or something.

Reply:
Should I fix all of these and make another commit?